src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    enum Colors {White, Yellow, Orange, Red, Magenta, Green, Blue, Pink, Black}
+
+    public static String getRandomColor() {
+        Random random = new Random();
```

Comment:
lets make Random as class level variable, not to create new instance of Random class every time we call getRandomColor() method